templates/helm/values.yaml.tpl

```diff
@@ -85,3 +85,7 @@ serviceAccount:
   name: {{ .ServiceAccountName }}
   annotations: {}
     # eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME
+
+driftRemediation:
+  defaultResyncPeriod: 0
+  configuration: '{}'
```

Comment:
What was your thought on this configuration option? Is it intended to support resource-specific resync periods? If so, perhaps we should call it `resources` and make it an `object` type instead of a `string` type?

Reply:
Changed to be an object